src/App.tsx

```diff
@@ -1,24 +1,107 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { HeaderTodoApp } from './components/HeaderTodoApp';
+import { MainTodoApp } from './components/MainTodoApp';
+import {
+  addTodo, deleteTodo, getTodos,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { FooterTodoApp } from './components/FooterTodoApp';
+import { Filters } from './types/Filters';
```

Comment:
Add `filters` to the  PR
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/91411692/6d4e1987-e613-4fda-81cd-13535855e211)